package/MDAnalysis/make_bundle.py

```diff
@@ -52,39 +54,45 @@ def make_bundle(*args, **kwargs):
     # TODO - use Group
     # TODO - careful about overwriting existing sims/category values
 
-    names = kwargs.get('names', map(str, range(len(args[0]))))
-    if len(args) == 2:
+    if not names:
+        # if names not provided, use the index for name
+        names = map(str, range(len(args[0])))
+    if not isinstance(simulations, list): 
```

Comment:
Yeah, that makes sense. I mainly stuck this is to catch errors if we couldn't index `simulations` + later errors if it was just a string - not sure if there's a concise way to check that it's a non-string indexable?